error in downloading boost

Hi, I tried to build the latest ogs but failed due to the below error. 

```
[  0%] Performing download step (download, verify and extract) for 'Boost'
-- downloading...
     src='http://sourceforge.net/projects/boost/files/boost/1.56.0/boost_1_56_0.tar.bz2/download'
     dst='/home/localadmin/ogs/ogs6/ogs6-nw-merged/BuildAllRelease/External/boost/src/boost_1_56_0.tar.bz2'
     timeout='none'
-- [download 100% complete]
CMake Error at Boost-stamp/download-Boost.cmake:27 (message):
  error: downloading
  'http://sourceforge.net/projects/boost/files/boost/1.56.0/boost_1_56_0.tar.bz2/download'
  failed

    status_code: 1
    status_string: "Unsupported protocol"
    log: Hostname was NOT found in DNS cache
    Trying 216.34.181.60...

  Connected to sourceforge.net (216.34.181.60) port 80 (#0)

  GET /projects/boost/files/boost/1.56.0/boost_1_56_0.tar.bz2/download
  HTTP/1.1

  User-Agent: curl/7.38.0

  Host: sourceforge.net

  Accept: */*

  

  HTTP/1.1 301 Moved Permanently

  Server nginx is not blacklisted

  Server: nginx

  Date: Tue, 01 Mar 2016 13:22:42 GMT

  Location:
  https://sourceforge.net/projects/boost/files/boost/1.56.0/boost_1_56_0.tar.bz2/download


  X-Content-Type-Options: nosniff

  Content-Length: 386

  Content-Type: text/html; charset=UTF-8

  Via: 1.1 wsa.ufz.de:80 (Cisco-WSA/9.0.1-162)

  Connection: keep-alive

  

  Ignoring the response-body

  <html>

   <head>
    <title>301 Moved Permanently</title>
   </head>
   <body>
    <h1>301 Moved Permanently</h1>
    The resource has been moved to <a href="https://sourceforge.net/projects/boost/files/boost/1.56.0/boost_1_56_0.tar.bz2/download">https://sourceforge.net/projects/boost/files/boost/1.56.0/boost_1_56_0.tar.bz2/download</a>;

  you should be redirected automatically.

  

  

   </body>

  </html>Connection #0 to host sourceforge.net left intact

  Issue another request to this URL:
  'https://sourceforge.net/projects/boost/files/boost/1.56.0/boost_1_56_0.tar.bz2/download'


  Protocol "https" not supported or disabled in libcurl

  Closing connection -1

  



make[2]: *** [External/boost/src/Boost-stamp/Boost-download] Error 1
make[1]: *** [CMakeFiles/Boost.dir/all] Error 2
make: *** [all] Error 2
```

--- a/scripts/cmake/ThirdPartyLibVersions.cmake
+++ b/scripts/cmake/ThirdPartyLibVersions.cmake
@@ -1,6 +1,6 @@
 set(OGS_BOOST_VERSION 1.56.0)
 string(REPLACE "." "_" OGS_BOOST_VERSION_UNDERLINE ${OGS_BOOST_VERSION})
-set(OGS_BOOST_URL "http://sourceforge.net/projects/boost/files/boost/${OGS_BOOST_VERSION}/boost_${OGS_BOOST_VERSION_UNDERLINE}.tar.bz2/download")
+set(OGS_BOOST_URL "http://opengeosys.s3.amazonaws.com/ogs6-lib-sources/boost_${OGS_BOOST_VERSION_UNDERLINE}.tar.bz2")
 set(OGS_BOOST_MD5 "a744cf167b05d72335f27c88115f211d")
 
 set(OGS_EIGEN_URL "http://opengeosys.s3.amazonaws.com/ogs6-lib-sources/eigen-3.2.5.tar.gz")

